wireapp/kalium
# chore: bump AVS version (AR-2944)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

Previous AVS versions were not handling properly the destructor of the calling object.

### Solutions

Bump AVS version to `9.1.12` so it handles the destructor of calling object properly also resolving the calling freeze we have been facing along with fixes [from this PR](https://github.com/wireapp/wire-android-reloaded/pull/1496)

### Dependencies (Optional)

[AVS lib](https://github.com/wireapp/wire-avs) v9.1.12


### Testing

#### How to Test

Use this Kalium branch along with latest develop on AR and make sure calling works (Incoming and outgoing / Video + Audio)
